scripts/portability.sh

```diff
@@ -28,6 +37,11 @@ fi
 # And ones only gcc produces
   CFLAGS+=" -Wno-restrict -Wno-format-overflow"
 
+case "$(uname)" in
+'NetBSD'|'OpenBSD')
+  unset ASAN;;
```

Comment:
:confused: does OpenBSD really have issues with address sanitation? Please explain for those of us who are uninitiated. (:pray: with small, gentle words please)

Reply:
:bow: Thanks for the explanation, I was unaware.

:speak_no_evil: and I can almost hear the musl libc devs strictly claiming the lack of a check here is a bug too . :roll_eyes:

perhaps consider a comment:
```suggestion
  #TODO: add compiler check for '-fsanitize=address' (e.g., $CC -h 2>1 | grep 'args')
  unset ASAN;; # otherwise assume no compiler support by default
```